tests/e2e/sse2/psllq.c

```diff
@@ -0,0 +1,32 @@
+#include <stdint.h>
```

Comment:
In addition to these tests could you see if qemu-test.c has a psllq/paddq test you can uncomment?